vulcanai2/models/basenetwork.py

```diff
@@ -281,6 +285,19 @@ def _create_network(self):
         :return: None
         """
         pass
+    
+    def forward(self, *inputs):
+        """Perform a forward pass through the module/modules."""
+
+        if isinstance(self.input_network, list):
+            outputs = []
+            for module in self.input_network:
+                outputs.append(module._forward(inputs))
+            network_output = torch.cat(outputs, 1)
+        else: 
+            network_output = self._forward(inputs)
+            
```

Comment:
here we would feed both through something like torch.cat((x,y),1)?

Reply:
not for both we don't need to concatenate when we have only one input.